### PR TITLE
Update orders model (closes #25).

### LINKED ISSUE #25: Flatten out Orders model.
Refactor it to more flat structure instead of two big objects. And probably not all fields from those two objects are needed to be saved?

### PR DESCRIPTION
**Pull request by g1st**
Selected what properties to save in `orders` collection and flattened out the model itself.

### CODE REVIEW

File: api/models/orders.js [hunk 1]
```diff
@@ -2,11 +2,21 @@ const mongoose = require('mongoose');
...
+    address2: { type: String, trim: true },
+    city: { type: String, trim: true },
+    country: { type: String, trim: true },
+    client_ip: { type: String, trim: true },
+    transaction_id: { type: String, trim: true },
+    receipt: { type: String, trim: true },
+    amount_paid: { type: Number },
+    source: { type: Object },
+    purchaseDetails: { type: Object },
+    additional_info: { type: String, trim: true }
```
Comment (nikrb) [inline api/models/orders.js]: LGTM
just a quick comment ... shouldn't the user details be in a user table and referenced from the order? I can see the possible need for a different shipping address, but shouldn't this still be in user table in an array of shipping addresses?

**Review by nikrb — DISMISSED** [review]
LGTM

File: api/models/orders.js (hunk 1 above)
Comment (g1st) [reply api/models/orders.js]: Agree with you. But we do not have users table or any data stored for a specific user, I'm saving every order as separate purchase (document) in orders collection. Probably will fix it as you suggested, thank you!

**Review by zurda — APPROVED** [review]

**Comment by zurda** [comment]
LGTM!

**Pull request merged**